app/services/stackmap_location_factory.rb

```diff
@@ -28,7 +28,7 @@ def excluded?(library)
         'Online',
         'Rare Books and Special Collections',
         'ReCAP',
-        'Lewis Library' # Temporarily inaccessible
```

Comment:
@kevinreiss did you mean to remove Lewis?

Reply:
Yes, their stacks are accessible again after the flood a bit ago.